packages/formation/sass/modules/_m-back-to-top.scss

```diff
@@ -0,0 +1,73 @@
+//=============================
+// Up To Top Button Styles
+//=============================
+#top-button-container {
```

Comment:
How do you feel about `back-to-top-container` to avoid potential namespace collision.